lamindb/models/history.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+from uuid import UUID
+
+# from .record import User --> only needed for ForeignKey
+# from django.contrib.contenttypes.models import ContentType --> only needed for ForeignKey
```

Comment:
I think it might be less hassle to not actually work with foreign key constraints in this context; also less space -- the constraints and indexes are the most costly